Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -80,6 +81,8 @@ public class FieldNpc : Actor<Npc> {
 
     public override Stats Stats { get; }
     public int TargetId = 0;
+    public string AiName { get; private set; }
```

Comment:
Is this needed? We can just get it from AiMetadata instead. Seeing as that's nullable and AiName is not may, it not be safe.

Reply:
The AI script can be changed, and that is used for pets to switch between combat mode and out of combat. I'm not sure of any specific cases outside that yet, but it would also be good for debugging purposes to be able to set it on the fly.
`<v key="PetBattleAiPath" value="Pet/AI_DefaultPetBattle.xml" />`
AI_DefaultPetTaming is the one that is used for pets out of combat and completely lacks any ability to fight, while AI_DefaultPetBattle is used exclusively for combat.

Reply:
But it's still redundant because if you are changing `AiName` here, you are also changing `AiMetadata` and `AiMetadata` already contains `Name`

Reply:
Ah now I see. I'll change it to a getter only using AiMetadata then